src/helpers/substrate.ts

```diff
@@ -1,5 +1,22 @@
 import { EventRecord } from '@polkadot/types/interfaces';
 import { ApiPromise } from '@polkadot/api';
+import Big from 'big.js';
+import { PalletBalancesAccountData } from '@polkadot/types/lookup';
+
+/// This function is used to get the frozen account balance from the account data.
+/// Some types define the frozen balance as `frozen` and others as `miscFrozen` and `feeFrozen`.
+export function getFrozenAccountBalance(balance: PalletBalancesAccountData) {
+  const { miscFrozen, feeFrozen } = balance;
+  const frozen = balance.get('frozen');
+  if (frozen !== undefined) {
```

Comment:
Can we just `if(frozen)`?
Or do we need to check for null, empty string, false or 0?